server/sonar-server/src/main/java/org/sonar/server/computation/step/ComputationSteps.java

```diff
@@ -41,6 +41,7 @@
       ParseReportStep.class,
 
       // Persist data
+      ComputeAndPersistDaysSinceLastCommitStep.class,
```

Comment:
I would rename the class "PersistDaysSinceLastCommitStep", no need to specify the "compute" word from my point of view